Assets/Scripts/GameManagement/GameManager.cs

```diff
@@ -2,21 +2,39 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
+using UnityEngine.EventSystems;
 using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.UI;
 
 public class GameManager : MonoBehaviour
 {
     public static bool launchToMainMenu = false; //CHANGE THIS DEPENDING ON WHETHER YOU WANT
                                                  //TO LAUNCH TO MAIN MENU OR TO YOUR CURRENT SCENE
     public static GameObject escMenu;
 
+    public static GameObject gameCanvas;
+
+    private static GameObject eventSystem;
+
+    private static GameObject inventory;
+
     [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.BeforeSceneLoad)]
     static void OnBeforeSplashScreen()
     {
         GameObject gameObject = Instantiate(new GameObject("GameManager"));
         DontDestroyOnLoad(gameObject);
         gameObject.AddComponent<GameStateManager>();
+
+        eventSystem = Instantiate(Resources.Load("Prefabs/UI/EventSystemPrefab")) as GameObject;
+        eventSystem.gameObject.transform.SetParent(gameObject.transform);
+
+        gameCanvas = Instantiate(Resources.Load("Prefabs/UI/GameCanvas")) as GameObject;
+        gameCanvas.gameObject.transform.SetParent(gameObject.transform);
+
         escMenu = Instantiate(Resources.Load("Prefabs/UI/EscMenuPrefab")) as GameObject;
-        escMenu.gameObject.transform.parent = gameObject.transform;
+        escMenu.gameObject.transform.SetParent(gameCanvas.transform);
```

Comment:
currently, the escmenu isn't scaled to the screen correctly, this could be the culprit

Reply:
This isn't happening for me, under what circumstances are you getting this?